execution-engine/contracts/hdac-system/client-api-proxy/src/client_api/mod.rs

```diff
@@ -109,6 +115,24 @@ impl Api {
                     .unwrap_or_revert_with(ApiError::InvalidArgument);
                 Api::Redelegate(src_validator, dest_validator, amount)
             }
+            method_names::proxy::VOTE => {
+                let dapp: Key = runtime::get_arg(1)
+                    .unwrap_or_revert_with(ApiError::MissingArgument)
+                    .unwrap_or_revert_with(ApiError::InvalidArgument);
+                let amount: U512 = runtime::get_arg(2)
+                    .unwrap_or_revert_with(ApiError::MissingArgument)
+                    .unwrap_or_revert_with(ApiError::InvalidArgument);
+                Api::Vote(dapp, amount)
+            }
+            method_names::proxy::UNVOTE => {
+                let dapp: Key = runtime::get_arg(1)
+                    .unwrap_or_revert_with(ApiError::MissingArgument)
+                    .unwrap_or_revert_with(ApiError::InvalidArgument);
+                let amount: U512 = runtime::get_arg(2)
```

Comment:
For api call, I think also need to change.
```
let amount: Option<U512>
```

Reply:
67e1226e0c1df3895034547e297d3f7edbeb8b4c